### PR TITLE
add paddle OCRNet

### PR DESCRIPTION
**Pull request by eaidova**
add ocrnet-hrnet-w48-paddle model to demonstrate PaddlePaddle framework support in OpenVINO

### CODE REVIEW

**Comment by eaidova** [comment]
Jenkins please retry a build

File: models/public/ocrnet-hrnet-w48-paddle/model.yml [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+    size: 3982
+    checksum: fbb592915c00af2d466847ffdf6eb71f16a49b261e13da5c4c71a9601e88e1eab18c1c554b27ee76ee74362938bd9423
+    source: https://raw.githubusercontent.com/PaddlePaddle/PaddleSeg/83a419ff92c54b01257c75d75a435df6d75e7515/export.py
+postprocessing:
+  - $type: unpack_archive
+    format: zip
+    file: paddleseg-2.3.0-py3-none-any.whl
+  # skip backbone downloading for weights intialization
+  - $type: regex_replace
+    file: paddleseg\models\backbones\hrnet.py
```
Comment (eaidova) [inline models/public/ocrnet-hrnet-w48-paddle/model.yml:57]:
```suggestion
    file: paddleseg/models/backbones/hrnet.py
```

**Comment by eaidova** [comment]
jenkins please retry a build

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -38,7 +39,7 @@ Examples of model names:
 
 ### Files Location
 
-Place your files as shown in the table below:
+Place your filesas shown in the table below:
```
Comment (eaidova) [inline CONTRIBUTING.md:42]:
```suggestion
Place your files as shown in the table below:
```

File: models/public/ocrnet-hrnet-w48-paddle/pre-convert.py [hunk 3]
```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env python3
+
+# Copyright (c) 2021 Intel Corporation
```
Comment (Anna-Mironova) [inline models/public/ocrnet-hrnet-w48-paddle/pre-convert.py:3]:
```suggestion
# Copyright (c) 2022 Intel Corporation
```

**Review by jkamelin — APPROVED** [review]

**Review by Anna-Mironova — APPROVED** [review]

**Review by vladimir-dudnik — APPROVED** [review]

**Comment by vladimir-dudnik** [comment]
Jenkins please retry a build

**Comment by eaidova** [comment]
Jenkins please retry a build

**Pull request merged**